packages/graphql/tests/integration/issues/2396.int.test.ts

```diff
@@ -0,0 +1,859 @@
+                                        },
+                                        area: 111,
+                                        estateType: "APARTMENT",
+                                        floor: 1,
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+            },
+            {
+                price: 250000,
+                valuation: {
+                    create: {
+                        node: {
+                            estate: {
+                                create: {
+                                    node: {
+                                        address: {
+                                            create: { node: { postalCode: { create: { node: { number: "13001" } } } } },
+                                        },
+                                        area: 77,
+                                        estateType: "APARTMENT",
+                                        floor: 2,
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+            },
+            {
+                price: 210000,
+                valuation: {
+                    create: {
+                        node: {
+                            estate: {
+                                create: {
+                                    node: {
+                                        address: {
+                                            create: { node: { postalCode: { create: { node: { number: "13001" } } } } },
+                                        },
+                                        area: 62,
+                                        estateType: "APARTMENT",
+                                        floor: 3,
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+            },
+            {
+                price: 279000,
+                valuation: {
+                    create: {
+                        node: {
+                            estate: {
+                                create: {
+                                    node: {
+                                        address: {
+                                            create: { node: { postalCode: { create: { node: { number: "13001" } } } } },
+                                        },
+                                        area: 48,
+                                        estateType: "APARTMENT",
+                                        floor: 3,
+                                    },
+                                },
+                            },
+                        },
+                    },
```

Comment:
I know we usually try to replicate the user example as close as possible, but couldn't we replicate the same problem with a smaller dataset?

Reply:
Hahahaha, perhaps, is there any major issue given that it's an exact replica? It is also cleaned up following execution.